packages/itwin/property-grid/package.json

```
@@ -46,7 +46,7 @@
     "lint:stylelint": "stylelint \"./src/**/*.scss\"",
     "lint:fix": "npm run lint:eslint -- --fix && npm run lint:stylelint -- --fix",
     "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
-    "test:dev": "cross-env NODE_OPTIONS=\"--import ../../../node-hooks/ignore-styles/register.cjs\" mocha --enable-source-maps --config ./.mocharc.json",
+    "test:dev": "node --enable-source-maps --import ../../../node-hooks/ignore-styles/register.cjs ./node_modules/mocha/bin/mocha.js --config ./.mocharc.json",
```

Can we run tests like this?
```suggestion
    "test:dev": "cross-env NODE_OPTIONS=\"--enable-source-maps --import ../../../node-hooks/ignore-styles/register.cjs\" mocha --config ./.mocharc.json",
```

---

yes, the change is only necessary if we use node mocks, which we don't for property grid tests